package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "ether-system-react",
+  "name": "nautilus-design-system-react",
```

Comment:
This is the name it's going to appear as in npm, so just make sure it's what you want 😄

Reply:
Pretty sure this changes in the next PR—I'm using `nautilus-system` right now, but  I'm open to something more elegant. `nautilus-react` maybe?

Reply:
`nautilus-react` works for me. 👍🏻